decentraland/unity-renderer
# feat: channels limit error

## What does this PR change?

Implementation of specific ways to show an error when the user tries to join a channel but already has reached the limit of available channels to join.

## How to test the changes?

1. Join or create 5 channels from `/join` command or create channel window
2. Go to create channel window
3. Try to create a channel
4. An error displaying that limit reached should be displayed
5. Type in chat `/join anychannel` command
6. A modal displaying that limit reached should be displayed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
